app/src/main/java/com/github/orkest/View/NavigationBar.kt

```diff
@@ -79,7 +82,9 @@ class NavigationBar {
                     composable("HomePage") { FeedActivity() }
                     composable("SearchPage") { SearchUserView.SearchUi(viewModel = viewModel) }
                     composable("PlaylistPage") { Text(text = "Playlist tab") }
-                    composable("ProfilePage") { ProfileTopInterface(viewModel = ProfileViewModel(user = currentUser)) }
+                    composable("ProfilePage") {
+                            ProfileActivityScreen(ProfileActivity(), viewModel = ProfileViewModel("JohnDoe"))
+                    }
                 }
```

Comment:
Why did you change the ProfileViewModel parameter ? Do your tests not pass without that change ?